test/ui/home/home_view_model_test.dart

```diff
@@ -144,13 +154,62 @@ void main() {
       expect(
           viewModel.stream,
           emitsInOrder([
-            const HomeViewState.loading(),
             const HomeViewState.error('Get Profile Error'),
             const HomeViewState.error('Get Surveys Error'),
             const HomeViewState.success(),
           ]));
 
       container.read(homeViewModelProvider.notifier).init();
     });
+
+    test('When calling signOut success, it emits Success state', () {
+      when(mockSignOutUseCase.call()).thenAnswer((_) async => Success(null));
+
+      expect(
+        container.read(signOutStream.future).asStream(),
+        emits(null),
+      );
+
+      // FIXME: This test is not working due to stream closed after first emitted value
+      // expect(
+      //   container.read(loadingIndicatorStream.future).asStream(),
+      //   emitsInOrder(
+      //     [
+      //       true,
+      //       false
+      //     ],
+      //   ),
+      // );
```

Comment:
@hoangnguyen92dn the issue might come from `loadingIndicatorStream.future`, I try to change it to `loadingIndicatorStream.stream` and it works correctly 😅 But `.stream` is deprecated, I will investigate this issue more. Let's comment it for now 👍